examples/active_support_helper.rb

```diff
@@ -0,0 +1,30 @@
+module ActiveRecord
+  class Base
+  end
+end
+
+module ActiveSupport
+  class TestCase < Minitest::Test
+    def setup(*args, &block)
+    end
+
+    # Remove describe method if present
+    class << self
+      remove_method :describe
+    end if self.respond_to?(:describe) &&
+        self.method(:describe).owner == ActiveSupport::TestCase
+
+    # Add spec DSL
+    extend ::Minitest::Spec::DSL
+
+    if defined?(ActiveRecord::Base)
```

Comment:
Yeah so the debate I was having was how much of minitest-rails to essentially duplicate and whether there was any value in keeping it looking like minitest-rails.  After I did that I was thinking that perhaps just simplifying it down to only the core basics that we need.  I was just thinking that it's tricky because we're going to have to distantly follow minitest-rails changes without an explicit coupling.